### PR TITLE
sdk 047

### PR DESCRIPTION
**Pull request by faddat**
Hi, @alpe -- here's a PR into the 47 branch you're working on.

If you'd rather not get these, please let me know.

Cheers!

-Jacob

Thus far, this PR:

* fixes lint issues
* fixes benchmark tests

### CODE REVIEW

**Comment by alpe** [comment]
Thanks Jacob! I appreciate your help. It is easy to miss some changes so any help is appreciated but let's stick with official repos. We are in close contact with the IBC team, too to get any issues out of our way. Status of inter-tx is one of them.

**Comment by faddat** [comment]
totally :).   think we've got a PR into ibc-go for the ibc, as I fully agree re: official repos.  Basically that replace is there so we could make progress.

File: app/ante.go [hunk 1]
```diff
@@ -39,11 +39,6 @@ func NewAnteHandler(options HandlerOptions) (sdk.AnteHandler, error) {
 		return nil, sdkerrors.Wrap(sdkerrors.ErrLogic, "tx counter key is required for ante builder")
 	}
 
-	sigGasConsumer := options.SigGasConsumer
```
Comment (alpe) [inline app/ante.go:42]: 👍 good eye

File: x/wasm/ibctesting/chain.go [hunk 2]
```diff
@@ -497,8 +497,8 @@ func (chain *TestChain) CreateTMClientHeader(chainID string, blockHeight int64,
 	// MakeCommit expects a signer array in the same order as the validator array.
 	// Thus we iterate over the ordered validator set and construct a signer array
 	// from the signer map in the same order.
-	signerArr := make([]tmtypes.PrivValidator, len(tmValSet.Validators))
-	for i, v := range tmValSet.Validators {
+	signerArr := make([]tmtypes.PrivValidator, len(tmValSet.Validators)) //nolint:staticcheck // TODO: determine if there's really a possible nil pointer dereference
```
Comment (alpe) [inline x/wasm/ibctesting/chain.go:500]: @faddat Are you going to follow up on this? Let's not add more todos in code

File: .circleci/config.yml [hunk 3]
```diff
@@ -61,7 +61,7 @@ jobs:
 
   lint:
     docker:
-      - image: golangci/golangci-lint:v1.46.2
```
Comment (alpe) [inline .circleci/config.yml:64]: This should go to main instead.

File: app/app.go [hunk 4]
```diff
@@ -127,9 +127,9 @@ import (
 
 	// Note: please do your research before using this in production app, this is a demo and not an officially
 	// supported IBC team implementation. It has no known issues, but do your own research before using it.
-	// intertx "github.com/cosmos/interchain-accounts/x/inter-tx"
```
Comment (alpe) [inline app/app.go:130]: I will keep inter-tx commented out until official supported versions are available. This is just a TODO

**Review by alpe — COMMENTED** [review]
Thank you for your submission again! I have cherry picked some commits to my development branch. I appreciate the help!  🏅

File: x/wasm/ibctesting/chain.go (hunk 2 above)
Comment (faddat) [reply x/wasm/ibctesting/chain.go:500]: PS: wdyt on this one?

Now that I'm looking at it, the TODO is def because I wasn't real sure at all.

**Comment by alpe** [comment]
Closing this in favour of #1136 . I would be happy to get your review

**Pull request closed without merging**